src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/resource/CsvDataExtractionController.kt

```diff
@@ -104,6 +104,13 @@ class CsvDataExtractionController(private val service: CsvDataExtractionService)
     "video-links-by-probation-booking-date-from-${startDate.toIsoDate()}-for-$days-days.csv",
   ) { os -> service.probationBookingsByBookingDateToCsv(inclusiveFromDate = startDate, exclusiveToDate = startDate.plusDays(days), os) }
 
+  @GetMapping(path = ["/prison-room-configuration-data"])
+  @Operation(description = "Return room decoration data for all active prisons in BVLS.")
+  @PreAuthorize("hasAnyRole('BOOK_A_VIDEO_LINK_ADMIN')")
+  fun downloadPrisonRoomConfigurationData(): ResponseEntity<StreamingResponseBody> = streamedCsvResponse(
+    "prison-room-configuration.csv",
```

Comment:
I know it's not specified in the requirements but perhaps consider appending a date/timestamp to the filename.

Reply:
Done